wies/projects/templates/colleague_list.html

```diff
@@ -1,6 +1,10 @@
 {% extends "base.html" %}
 {% load static %}
 
+{% block extra_head %}
+<link rel="stylesheet" href="{% static 'css/placements.css' %}">
```

Comment:
ik zou denken dat placements.css alleen css bevat specifiek voor placements. als we hier in colleagues vergelijkbare css nodig hebben dan zou ik die verplaatsen naar custom.css en dan wordt dit automatisch meegenomen